src/tox/provision.py

```diff
@@ -75,6 +75,14 @@ def provision(state: State) -> int | bool:
 
     def add_tox_requires_min_version(requires: list[Requirement]) -> list[Requirement]:
         min_version: Version = state.conf.core["min_version"]
+        # If own version can be a development one or a pre-release, we need to
+        # to only use its base_version for requirements, or pip will never be
+        # able to find a version that is compatible with the requirement.
+        # See https://github.com/tox-dev/tox/issues/2414
```

Comment:
Remove this line. I don't believe in linking to upstream issues. If there's something I need to know, explain it here or otherwise not important enough.